package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "onedrive-api",
-  "version": "0.3.4",
+  "version": "0.4.5",
```

Comment:
This should be `"0.4.0"`, after updating minor version we are resetting patch version

Reply:
Understood